src/Symfony/Component/Form/Extension/Core/Type/DateTimeType.php

```diff
@@ -317,6 +318,8 @@ public function configureOptions(OptionsResolver $resolver)
 
             return '';
         });
+
+        $resolver->setAllowedTypes('input_format', 'string');
```

Comment:
https://github.com/symfony/symfony/blob/a6a1be803d7e757cbfdf43b7d819e4c8a9d4e7d9/src/Symfony/Component/Form/Extension/Core/Type/DateTimeType.php#L298-L299
As we are going to do with the `time_widget` option, I wonder if we should throw an exception when the` input_format` is passed and the value of `input` is not strictly equal to` string'`.